### PR TITLE
rails 1.4.2

### PR DESCRIPTION
**Pull request by lcoombe**
- [x] Have you followed the [guidelines for contributing](https://github.com/brewsci/homebrew-bio/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/brewsci/homebrew-bio/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source FORMULA`, where `FORMULA` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict FORMULA` (after doing `brew install FORMULA`)?

-----

### CODE REVIEW

**Comment by sjackman** [comment]
@warrenlr For future releases, I'd recommend using the versioning scheme described by https://semver.org. For a minor bug fix, use release 1.4.2 rather than 1.4.1.1.

**Comment by warrenlr** [comment]
You caught me!  I could still change it... please advise (I just thought the changes were not worthy of a 0.0.1 increment)

File: Formula/rails.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 class Rails < Formula
-  # cite L_Warren_2016: "https://doi.org/10.21105/joss.00116"
+  # cite Warren_2016: "https://doi.org/10.21105/joss.00116"
```
Comment (sjackman) [inline Formula/rails.rb]: I've reported this incorrect Crossref metadata to JOSS at https://github.com/openjournals/joss/issues/521
cc @warrenlr

Comment (lcoombe) [reply Formula/rails.rb]: Great! Yeah I just noticed it so figured I'd fix it manually

**Comment by sjackman** [comment]
> You caught me! I could still change it... please advise (I just thought the changes were not worthy of a 0.0.1 increment)

I find sticking to a X.Y.Z version scheme really helpful. If it's easy enough to change, please do.

**Comment by lcoombe** [comment]
Yes I can update this PR using a new release tarball since I haven't published the bottles yet.

**Comment by warrenlr** [comment]
ok, then I'll make changes to the release version

**Comment by warrenlr** [comment]
done, sorry for the extra work on your end

**Comment by sjackman** [comment]
No worries at all, René. Thanks for the fix

**Comment by lcoombe** [comment]
No worries - it's an easy change on my end! Just pushed the updated version :)

**Comment by sjackman** [comment]
Thanks, Lauren! 🌟

**Pull request closed without merging**